From 7414bb88e33064bf28629ee72c4dfa00a169ef93 Mon Sep 17 00:00:00 2001
From: Lucas RAVAGNIER <ravagnierlucas@gmail.com>
Date: Tue, 14 Jan 2025 16:15:29 +0100
Subject: [PATCH 9/9] fix(curl): resolve TLS issue caused by restrictive
 configuration

Using a `.curlrc` file restricts the use of openssl encryption.
This restriction blocks the ability to deploy an XOA because
`xoa.io` website requires newer certificates.
This patch therefore aims to add the same encryption from the xoa.io,
making deployment easier for users while maintaining secure communications.

Signed-off-by: Lucas RAVAGNIER <ravagnierlucas@gmail.com>
---
 src/common/root/.curlrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/root/.curlrc b/src/common/root/.curlrc
index c6113af..0726695 100644
--- a/src/common/root/.curlrc
+++ b/src/common/root/.curlrc
@@ -1,3 +1,3 @@
 # See https://curl.haxx.se/docs/ssl-ciphers.html for the NSS names
-ciphers = ecdhe_rsa_aes_256_sha_384,ecdhe_rsa_aes_256_gcm_sha_384,rsa_aes_256_cbc_sha_256,rsa_aes_128_cbc_sha_256
+ciphers = ecdhe_rsa_aes_256_sha_384,ecdhe_rsa_aes_256_gcm_sha_384,rsa_aes_256_cbc_sha_256,rsa_aes_128_cbc_sha_256,ECDHE-ECDSA-AES128-GCM-SHA256
 tlsv1.2
-- 
2.47.0

